### PR TITLE
[CIRCSTORE-113] Expiration unit tests can fail

### PR DESCRIPTION
**Pull request by opochepam**

### CODE REVIEW

File: src/main/java/org/folio/support/ExpirationTool.java [hunk 1]
```diff
@@ -1,13 +1,5 @@
 package org.folio.support;
 
-import java.text.SimpleDateFormat;
```
Comment (marcjohnson-kint) [inline src/main/java/org/folio/support/ExpirationTool.java]: Please can these changes be reset to use the ordering defined in https://issues.folio.org/browse/FOLIO-1667

File: src/test/java/org/folio/rest/api/RequestExpirationApiTest.java [hunk 2]
```diff
@@ -42,24 +42,13 @@ public void checkIdsAfterEach() {
...
-      new RequestRequestBuilder()
-      .hold()
-      .withId(id)
-      .withRequestExpiration(new DateTime(2017, 7, 30, 10, 22, 54, DateTimeZone.UTC))
-      .withItemId(itemId)
-      .withPosition(1)
-      .withStatus(OPEN_NOT_YET_FILLED)
-      .create(),
-      requestStorageUrl());
+    createEntity(new RequestRequestBuilder().hold().withId(id).withRequestExpiration(new DateTime(2017, 7, 30, 10, 22, 54,
```
Comment (marcjohnson-kint) [inline src/test/java/org/folio/rest/api/RequestExpirationApiTest.java]: Can you help me understand why these lines have been reformatted?

I personally find it much easier to interpret the relevant details of the records with the multi line layout.

Comment (kostyantyn-kh) [reply src/test/java/org/folio/rest/api/RequestExpirationApiTest.java]: reformatted after applying import ordering with Eclipse Code Formatter
reverting this part

**Comment by opochepam** [comment]
> ke significant formatting c

Please take a look at 3ab56bd1398e532ecc07e182dd0cfa1a7e0601bf.

Method `closeRequests` returns `List<Future>` and then `CompositeFuture` created based on this list . The list filled up inside `runOnContext` lambda and then `CompositeFuture` created based on the empty list. In other words the `closeRequests` returned `List<Future>` which hadn't beed filled up because of running on a context.

File: src/test/java/org/folio/rest/api/RequestExpirationApiTest.java (hunk 2 above)
Comment (kostyantyn-kh) [reply src/test/java/org/folio/rest/api/RequestExpirationApiTest.java]: reverted

File: src/test/java/org/folio/rest/api/RequestExpirationApiTest.java [hunk 3]
```diff
@@ -443,9 +445,7 @@ public void canExpireOpenAwaitingWithNoHoldShelfExpirationDate()
 
   private void expireRequests() throws InterruptedException, ExecutionException, TimeoutException {
     CompletableFuture<Void> expirationCompleted = new CompletableFuture<>();
-    ExpirationTool.doRequestExpiration(StorageTestSuite.getVertx(),
-      StorageTestSuite.getVertx().getOrCreateContext())
-      .setHandler(res -> expirationCompleted.complete(null));
+    ExpirationTool.doRequestExpiration(StorageTestSuite.getVertx()).setHandler(res -> expirationCompleted.complete(null));
```
Comment (marcjohnson-kint) [inline src/test/java/org/folio/rest/api/RequestExpirationApiTest.java]: Similarly, was this automatic reformatting? I personally find the multiple line version easier to comprehend.

Comment (kostyantyn-kh) [reply src/test/java/org/folio/rest/api/RequestExpirationApiTest.java]: i think so, reverted this

**Comment by hjiebsco** [comment]
@marcjohnson-kint, can we get this merged soon, so other PRs will not see unrelated build failures?

**Pull request merged**